.github/actions/pnpm-setup-action/action.yml

```diff
@@ -3,10 +3,10 @@ description: "Installs Node, pnpm, fetches cache (if exists), and runs pnpm inst
 runs:
   using: "composite"
   steps:
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js ${{ matrix.node-version || 18 }}
       uses: actions/setup-node@v3
       with:
-        node-version: ${{ matrix.node-version }}
```

Comment:
Add a fallback jobs where we don't declare a matrix (test, lint). Before it would fall back to 16!